modules/renter/download_test.go

```diff
@@ -20,6 +23,8 @@ type testHost struct {
 	// used to simulate real-world conditions
 	delay    time.Duration // transfers will take this long
 	failRate int           // transfers will randomly fail with probability 1/failRate
+
+	sync.Mutex
```

Comment:
makes me pause a bit. If your testing structures are that complex, you might need to add testing for your testing :P